CommunicationLayer/CommDeviceControl/UdpMessageForwarder.cpp

```diff
@@ -43,10 +47,6 @@ UdpMessageForwarder::~UdpMessageForwarder()
 
 void UdpMessageForwarder::forwardData(QByteArray data)
 {
-    qDebug() << "UdpMessageForwarder: Forwarding data";
-    const quint64 dataWritten = socket_.writeDatagram(data, ipAddress_, port_);
-    if (dataWritten != static_cast<quint64>(data.size()))
-    {
-        qWarning() << "UdpMessageForwader: Unable to forward data";
```

Comment:
There must be some case that data still can't be forwarded, this function should be able know when that happens and at least list a warning.